pkg/commands/catalog/copy/copy.go

```diff
@@ -89,18 +90,27 @@ func extractImageURLs(filePath string) ([]string, error) {
 
 // copyImagesToNewDomain parses the image list and copies the images to the new domain returning a list of images that were copied successfully
 func copyImagesToNewDomain(images []string, newImageURLs []string, arch string) ([]string, error) {
+	var err error
 	copied := make([]string, 0, len(images))
 	for i, theimage := range images {
 		newImageURL := newImageURLs[i]
-		err := image.Copy(theimage, newImageURL, arch, copy.CopyAllImages)
-		if err != nil {
-			log.Errorf("Error copying image %s: %s", theimage, err.Error())
-		} else {
-			log.Infof("Successfully copied image %s", theimage)
-			copied = append(copied, theimage)
+		for j := 0; j < 5; j++ {
```

Comment:
Instead of creating our homegrown retry logic, we should look at using appropriate retry logic from an existing third-party library in the long run. Search results for retry keyword https://pkg.go.dev/search?limit=100&m=symbol&q=retry

Reply:
We have existing retry functions: https://github.com/oracle-cne/ocne/blob/main/pkg/util/backoff.go

Reply:
switched to util.LinearRetry